src/components/modal/ModalDialog.js

```diff
@@ -116,8 +116,8 @@ class ModalDialog extends React.Component<Props> {
                     <h2 className="modal-title" id={`${this.modalID}-label`}>
                         {title}
                     </h2>
+                    <>{this.renderCloseButton()}</>
```

Comment:
I think we can remove the fragment component.

Reply:
The Fragment is there so it can render inside the title Div, otherwise the build breaks.

Reply:
I think omitting it should be fine since the method is returning a jsx button. This is how we currently have it on `master`:
<img width="252" alt="Screen Shot 2022-05-27 at 12 41 48 PM" src="https://user-images.githubusercontent.com/7311041/170779526-21ece666-053b-4a88-8755-ab18d0f015f0.png">

Reply:
Done